### PR TITLE
Fix clients

### PR DESCRIPTION
**Pull request by Seequick1**
Signed-off-by: Seequick1 <morsak@redhat.com>

### Type of change

- Bugfix

### Description

This PR just changing naming of clients. 
- Clients inside pods = internal 
- Clients in memory = external

### Checklist

- [ ] Make sure all tests pass

### CODE REVIEW

**Comment by Seequick1** [comment]
@strimzi-ci run tests

**Comment by Seequick1** [comment]
@strimzi-ci run tests

File: systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java [hunk 1]
```diff
@@ -498,12 +498,12 @@ void testSendMessagesPlainAnonymous() throws InterruptedException {
...
             ResourceManager.kubeClient().listPodsByPrefixInName(CLUSTER_NAME + "-" + Constants.KAFKA_CLIENTS).get(0).getMetadata().getName();
 
-        externalKafkaClient.setPodName(defaultKafkaClientsPodName);
+        internalKafkaClient.setPodName(defaultKafkaClientsPodName);
 
-        LOGGER.info("Checking produceed and consumed messages to pod:{}", externalKafkaClient.getPodName());
-        externalKafkaClient.checkProducedAndConsumedMessages(
-            externalKafkaClient.sendMessages(topicName, NAMESPACE, CLUSTER_NAME, messagesCount),
-            externalKafkaClient.receiveMessages(topicName, NAMESPACE, CLUSTER_NAME, messagesCount, CONSUMER_GROUP_NAME)
+        LOGGER.info("Checking produceed and consumed messages to pod:{}", internalKafkaClient.getPodName());
```
Comment (ppatierno) [inline systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java]: typo "produced"

**Comment by Seequick1** [comment]
@strimzi-ci run tests test_only

**Comment by Frawless** [comment]
Cherry-pick this to 0.17.x

**Pull request merged**